### PR TITLE
scaleOfMeasure

### PR DESCRIPTION
**Pull request by l00mi**

### CODE REVIEW

**Comment by l00mi** [comment]
@tpluscode i don't get the typechecking to run quickly.. can you please collect the rest? i do currently als use qudt:IntervalScale for everything which was temporal.

File: ui/src/components/ScaleOfMeasureIcon.vue [hunk 1]
```diff
@@ -7,7 +7,14 @@
...
 import { Term } from 'rdf-js'
-import { scale } from '@cube-creator/core/namespace'
+import { qudt } from '@tpluscode/rdf-ns-builders'
+
+const icons: Record<string, string> = {
+  [qudt.NominalScale.value]: 'list-ul',
+  [qudt.OrdinalScale.value]: 'list-ul',
+  [qudt.IntervalScale.value]: 'clock',
+  [qudt.RatioScale.value]: 'clock',
+}
```
Comment (tpluscode) [inline ui/src/components/ScaleOfMeasureIcon.vue:17]: Nominal/Ordinal unchanged

Both Interval and Ratio use clock icon? @l00mi

**Comment by l00mi** [comment]
@tpluscode i fixed the icons.. anything else missing?

**Comment by l00mi** [comment]
if all good please go ahead and merge for me.

**Comment by tpluscode** [comment]
> anything else missing?

so for now we only keep the 4 scale types?

**Comment by l00mi** [comment]
yes.. and i will start adding the other metadata for the other cases.

**Review by tpluscode — APPROVED** [review]

**Pull request merged**